pkg/actors/actors_test.go

```diff
@@ -1843,3 +1876,18 @@ func TestReentrancyStackLimit(t *testing.T) {
 	assert.Nil(t, resp)
 	assert.Error(t, err)
 }
+
+func TestPlacementSwitchIsNotTurnedOn(t *testing.T) {
+	testActorsRuntime := newTestActorsRuntimeWithoutPlacement()
+
+	t.Run("placement is empty", func(t *testing.T) {
+		assert.Nil(t, testActorsRuntime.placement)
+	})
+
+	t.Run("the store can still be initialized normally", func(t *testing.T) {
+		assert.NotNil(t, testActorsRuntime.store)
```

Comment:
why is the runtime store initialised if placement server is going to be disabled?
/cc @yaron2 @artursouza

Reply:
That's correct, @CrazyHZM please make sure the actor runtime and it's state store aren't initialized if disabled.
The state store should still be initialized, just not as an actor state store.

Reply:
@mukundansundar @yaron2 
The related logic has been adjusted and UT has been added.